vignettes/articles/tilt-company-match.Rmd

```diff
@@ -199,7 +199,7 @@ these conditions:
 
 ```{r}
 # Decided upon extensive experience with r2dii.match function and processes
-threshold <- 0.9
+threshold <- 0.75
```

Comment:
Qithout nowing the context of this article I do not think this is correct. In the readme there is 2 different thresholds now.
1.  one lower threshold for even including candidates: https://github.com/2DegreesInvesting/tilt.company.match/blob/8db1b9556359a29bb3e5f28cf52177633a004467/README.Rmd#L217
2. a threshold for including candidates as accepted matches which is still 0.9 and did not change: https://github.com/2DegreesInvesting/tilt.company.match/blob/8db1b9556359a29bb3e5f28cf52177633a004467/README.Rmd#L272